packages/ripple-nuxt-tide/modules/webform/conditional-logic.js

```diff
@@ -24,7 +26,7 @@ function testField (field, data) {
         }
         break
       default:
-        console.warn(`Form: State "${state}" is not supported.`)
+        logger.warn(`Form: State "${state}" is not supported.`, { label: 'Webform' })
```

Comment:
Should we be using the %s syntax for consistency?

Reply:
Yes, should. But based on the amount of changes, I feel leave it as it is will make the work done quicker. I will have a fix now anyway!